windows/win_feature.py

```diff
@@ -87,11 +87,24 @@
   tasks:
     - name: Install IIS
       win_feature:
-        name: "Web-Server"
-        state: absent
+        name: "Web-Server,Web-Mgmt-Tools,web-Default-Doc"
+        state: Present
```

Comment:
Should be lowercase to be consistent with what the module is expecting in https://github.com/trondhindenes/ansible-modules-core/blob/fca52e6f16a61756b7183aaa748aa9567e448410/windows/win_feature.ps1#L84